cadc-dali/src/main/java/ca/nrc/cadc/dali/tables/votable/VOTableWriter.java

```diff
@@ -287,19 +287,6 @@ protected void writeImpl(VOTableDocument votable, Writer writer, Long maxrec)
                 Element table = new Element("TABLE", namespace);
                 resource.addContent(table);
 
-                // Create the INFO element and add to the RESOURCE element.
```

Comment:
The more compact version using createInfo() is fine, but why move all the INFO elements to after the params and table data? 

There is a use for trailing INFO elements (error and overflow), but that is separate and explicit

Reply:
I don't remember why I moved it, maybe because I added the same statements to the VOTable root, so I put them side by side?